readme.md

```diff
@@ -76,5 +76,23 @@ The maximum number of messages to return. [AWS Documenation](http://docs.aws.ama
 
 The duration (in seconds) for which the call waits for a message to arrive in the queue before returning. [AWS Documenation](http://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_ReceiveMessage.html)
 
+#### `onLambda` (function: optional)
```

Comment:
Naming is difficult, can you share any background on this choice? I know the naming doesn't per se matter either. `onInvoke` `afterInvoke`?